data/Debian-family.yaml

```diff
@@ -1,2 +1,3 @@
 ---
 systemd::nspawn_package: 'systemd-container'
+systemd::journal_remote::package_name: 'systemd-journal-remote'
```

Comment:
Would be good to add an acceptance test to keep confirming all the package names are correct.

Reply:
right, bu was not able to create a succesfull test. I assume, I cannot start systemd-journal-remote service in the environment. I'm giving up now ;(
(see #482  for the commit trying ;))